### ISSUE TITLE
Status page does not show manager version

### ISSUE THREAD

**Issue by gdiazlo** [wazuh/wazuh#21568]
| Wazuh | Rev  | Browser                      |
|-------|------|------------------------------|
| 4.8.0 | alpha 2 | Chrome, Firefox, Safari, etc |

**Description**

The status page does not show the version.
The endpoint (/api/setup)  is returning the version, as in previous releases.

**Preconditions**
1.  Deploy canonica Wazuh environment

**Steps to reproduce**
1. Navigate to Server management -> status
2. Review Manager information

**Expected Result**

<img width="868" alt="image" src="https://github.com/wazuh/wazuh-dashboard-plugins/assets/2181918/e90b3368-d2e5-4ba8-a92d-00e63f3f734f">

**Actual Result**

<img width="874" alt="image" src="https://github.com/wazuh/wazuh-dashboard-plugins/assets/2181918/dafdc22b-e3c2-43f8-9bcd-005262d6dd8a">

**Comment by GGP1** [comment]
## Update

Removed the metadata file and obtained the information from other sources. 

We will discuss with the team whether we need to keep this information or not, if we decide to keep it, we should get the information like this.

**Comment by GGP1** [comment]
## Update

- Mocked the new function in the unit tests affected
- Cached the function result to execute it only once during the process lifetime
- Removed an import that is no longer used

Moving the issue on hold until we decide whether to keep the installation date information or not.

**Comment by GGP1** [comment]
## Update

Uploaded the results of the AIT tier 0-1 that are failing sporadically. Checked the other tests that failed to verify if they are related but they are not. Moving status to pending review
